setup.py

```diff
@@ -0,0 +1,27 @@
+import setuptools
+
+with open("README.md", "r", encoding="utf-8") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    # TODO: Decide name for package
+    name="TO BE DECIDED",
+    version="0.0.1",
+    author="Corey Koelewyyn",
+    author_email="Corey.Koelewyn@gmail.com",
+    description="A scheduling package for UVIC SENG499 capstone",
```

Comment:
nit: "A course scheduler for the software engineering program at UVic. Built for the SENG 499 Summer 2022 project."